ecl/hql/hqlutil.cpp

```diff
@@ -8809,6 +8815,25 @@ bool ConstantRowCreator::processFieldValue(IHqlExpression * optLhs, ITypeInfo *
             castValue->toMem(out.reserve(lenValue));
             return true;
         }
+    case type_varunicode:
+    {
+        if (sizeLhs == UNKNOWN_LENGTH)
+        {
+            void * target = out.reserve((lenValue+1)*sizeof(UChar));
+            castValue->toMem(target);
+        }
+        else
+        {
+            //Disabled for the moment to prevent the size of generated expressions getting too big.
+            if (sizeLhs > 40)
```

Comment:
I suspect the two should match. I'll open a separate jira to increase it (though I don't know what the impact is - can you explain?)